templates/mjml/default.mjml

```diff
@@ -1,12 +1,27 @@
 <mjml>
   <mj-head>
+    <mj-raw>
+      <style type="text/css">#outlook a { padding:0; }
+          .ReadMsgBody { width:100%; }
+          .ExternalClass { width:100%; }
+          .ExternalClass * { line-height:100%; }
```

Comment:
can you add some comments on what are these classes and where do they come from?